shelly/src/main/java/au/com/jtribe/shelly/Email.java

```diff
@@ -13,127 +11,153 @@
  * Represents an email to a person or group of people.
  */
 public final class Email {
-    private final Context context;
-
-    private final List<String> toList;
-    private final List<String> ccList;
-    private final List<String> bccList;
-    private String subject;
-    private String body;
-
-    Email(Context context) {
-        this.context = context;
-        this.toList = new ArrayList<>(4);   // arbitrary assume people won't be sending toList more than 4 people in most cases.
-        this.ccList = new ArrayList<>();    // In most cases these will probably be empty anyway
-        this.bccList = new ArrayList<>();
+
+  private final List<String> toList;
+  private final List<String> ccList;
+  private final List<String> bccList;
+  private String subject;
+  private String body;
+
+  Email() {
+    this.toList = new ArrayList<>(
+        4);   // arbitrary assume people won't be sending toList more than 4 people in most cases.
+    this.ccList = new ArrayList<>();    // In most cases these will probably be empty anyway
+    this.bccList = new ArrayList<>();
+  }
+
+  /**
+   * Adds email addresses that the email should be sent to. Email addresses will be added into the
+   * To field.
+   *
+   * @param to Urls that the email should be sent to.
+   * @return Object this method was called on for method chaining.
+   */
+  public Email to(String... to) {
```

Comment:
Annotate the method with `@CheckResult` and `@NonNull`
Annotate the parameter with `@NonNull`